tox.ini

```diff
@@ -77,6 +76,8 @@ deps =
     pytest==7.4.1
     # renovate: datasource=pypi
     pytest-operator==0.29.0
+    nats-py==2.4
```

Comment:
Shouldn't it be 2.4.0, same as in tests/integration/relation_tests/application-charm/requirements.txt?
Also, shouldn't there be `ops==2.6.0` also?